cmd/create/cluster/cmd.go

```diff
@@ -563,6 +579,11 @@ func run(cmd *cobra.Command, _ []string) {
 			"Any optional fields can be left empty and a default will be selected.")
 	}
 
+	// warn if sts flag is used
+	if cmd.Flags().Changed("sts") {
+		r.Reporter.Warnf("Cluster creation is with STS by default, and the --sts flag is no longer necessary.")
```

Comment:
Please change:
Cluster creation is with STS by default and the --sts flag is no longer necessary.